rasa/validator.py

```diff
@@ -64,9 +65,14 @@ def verify_intents(self, ignore_warnings: bool = True) -> bool:
 
         nlu_data_intents = {e.data["intent"] for e in self.intents.intent_examples}
 
-        for intent in self.domain.intents:
+        # Exclude the default intents that are always implicitly present
+        for intent in [
+            item
+            for item in self.domain.intents
+            if item not in constants.DEFAULT_INTENTS
+        ]:
```

Comment:
optional: We could also make this a property on `Domain` to avoid duplicating the logic

Reply:
Yup can definitely do! Do we have a general policy for code repetitions and when to de-duplicate them? My usual approach is to tackle those when code would otherwise be triplicated.

Reply:
> My usual approach is to tackle those when code would otherwise be triplicated.

I think that's a good rule of thumb. In this case I just felt like we'd need that in several places sooner or later and it felt quite "natural" to the Domain. I like the solution which you chose 💯